File: Strings/pagina06/3141/3141.php
<?php

$nome = readline();
list($a, $b, $c) = explode("/", readline());
list($d, $e, $f) = explode("/", readline());

$dia_atual = (int) $a; $mes_atual = (int) $b; $ano_atual = (int) $c;
$dia_nasci = (int) $d; $mes_nasci = (int) $e; $ano_nasci = (int) $f;

$idade = $ano_atual - $ano_nasci;

if($dia_atual == $dia_nasci && $mes_atual == $mes_nasci){
    echo "Feliz aniversario!\n";
}

if($mes_atual < $mes_nasci || ($mes_atual == $mes_nasci && $dia_atual < $dia_nasci)){
    $idade -= 1;
}

echo "Voce tem $idade anos $nome.\n";

File: Strings/pagina06/3141/3141.cpp
#include <iostream>

using namespace std;

int main(){

    string nome, data1, data2;
    int dia_atual, mes_atual, ano_atual;
    int dia_nasci, mes_nasci, ano_nasci;

    getline(cin, nome);
    scanf("%d/%d/%d", &dia_atual, &mes_atual, &ano_atual);
    scanf("%d/%d/%d", &dia_nasci, &mes_nasci, &ano_nasci);

    
    if(dia_nasci == dia_atual && mes_nasci == mes_atual){
        cout << "Feliz aniversario!" << endl;
    }

    int idade = ano_atual - ano_nasci;
    if(mes_atual < mes_nasci || (mes_atual == mes_nasci &&  dia_atual < dia_nasci)){
        idade -= 1;
    }

    cout << "Voce tem " + to_string(idade) + " anos " + nome + "." << endl;

    return 0;
}